pkg/skaffold/schema/latest/config.go

```diff
@@ -146,6 +148,24 @@ type TestCase struct {
 	StructureTests []string `yaml:"structureTests,omitempty"`
 }
 
+// Deployer contains properties shared between deployers
+type Deployer struct {
+	KubeContext string `yaml:"context,omitempty"`
+}
+
+// ParseContext returns the kubectl context to use for deployment
+func (d Deployer) ParseContext(kubeContext string) string {
```

Comment:
This doesn't really parse anything. Why not just `GetContext(..)` or even `Context(..)`?

Reply:
ParseContext logic should be outside the config - most likely in https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/runner/runner.go#L64